students/11th/yejikang/js/login.js

```diff
@@ -0,0 +1,18 @@
+const loginButton = document.querySelector('.loginBtn');
+const loginForm = document.querySelector('.loginForm');
+const userId=document.querySelector('.userId');
+const userPw = document.querySelector('.userPw');
+
+const init = (function () {
+    loginButton.disabled = true;
+})();
+
+function setEnabledButton() {
+     loginButton.disabled =  (userId.value && userPw.value)? false : true ; 
+}
+
+loginForm.addEventListener('keyup',setEnabledButton);
+
+loginButton.addEventListener('click', function(){
+     console.log(userId.value,userPw.value);   
+});
```

Comment:
```suggestion
loginForm.addEventListener('keyup',setEnabledButton);

loginButton.addEventListener('click', function() {
  console.log(userId.value, userPw.value);   
});
```
1. 띄어쓰기도 잘 해주셔야 코드 가독성이 좋아집니다
2. 테스트 끝난 console은 지워주세요
3. vscode에서 tab size 2칸으로 바꿔주세요

Reply:
넵 수정했습니다!

Reply:
네!